datadog_cluster_agent/assets/dashboards/datadog_cluster_agent_overview.json

```diff
@@ -392,1374 +912,948 @@
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "on_right_yaxis": false,
+                                    "queries": [
+                                        {
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "query": "sum:datadog.cluster_agent.admission_webhooks.library_injection_errors{$cluster, $namespace} by {reason}.as_count()"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "style": {
+                                        "palette": "dog_classic",
+                                        "line_type": "solid",
+                                        "line_width": "normal"
+                                    },
+                                    "display_type": "bars"
+                                }
+                            ],
+                            "yaxis": {
+                                "include_zero": true,
+                                "scale": "linear",
+                                "min": "auto",
+                                "max": "auto"
+                            },
+                            "markers": []
                         },
-                        "display_type": "bars"
-                    }
-                ],
-                "yaxis": {
-                    "include_zero": true,
-                    "scale": "linear",
-                    "min": "auto",
-                    "max": "auto"
-                },
-                "markers": []
-            },
-            "layout": {
-                "x": 4,
-                "y": 18,
-                "width": 4,
-                "height": 2
-            }
-        },
-        {
-            "id": 4404687275565510,
-            "definition": {
-                "title": "Certificate validity - hours left",
-                "type": "query_value",
-                "requests": [
+                        "layout": {
+                            "x": 4,
+                            "y": 6,
+                            "width": 4,
+                            "height": 2
+                        }
+                    },
                     {
-                        "conditional_formats": [
-                            {
-                                "comparator": "<",
-                                "hide_value": false,
-                                "palette": "red_on_white",
-                                "value": 72
+                        "id": 7089651226858956,
+                        "definition": {
+                            "title": "Successful validation attempts per minute by type",
```

Comment:
nit: is there a reason these graphs are named differently i.e. `Successful ... per minute by type` vs `Failed ... by type per minute`?
same applies for the `mutation attempts` graphs